bitnami/redis/templates/configmap-scripts.yaml

```diff
@@ -22,10 +24,35 @@ data:
             false
         fi
     }
+    retry_while() {
+        local cmd="${1:?cmd is missing}"
+        local retries="${2:-12}"
+        local sleep_time="${3:-5}"
+        local return_value=1
+        read -r -a command <<< "$cmd"
+        for ((i = 1 ; i <= retries ; i+=1 )); do
+            "${command[@]}" && return_value=0 && break
+            sleep "$sleep_time"
+        done
+        return $return_value
+    }
```

Comment:
We're loading these functions directly from the container in other solutions (e.g. Redis Cluster), see https://github.com/bitnami/charts/blob/master/bitnami/redis-cluster/templates/update-cluster.yaml#L88
@javsalgar what do you think? This will be incompatible with non-bitnami images, but it's cleaner.

Reply:
I added it because there are other already copied here.

Reply:
If it's just loading the helper functions, I agree on having them directly from the container. But I wouldn't move the sentinel logic to the container. So answering to the question, I'm fine with changing it to a `. /opt/bitnami/scripts/libwhatever.sh`